blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/Instrumentation.java

```diff
@@ -173,4 +168,18 @@ public static void markMetricsWithShortDelayReceived() {
     public static void markMetricsWithLongDelayReceived() {
         metricsWithLongDelayReceived.mark();
     }
+
+    public static Meter getIngestedMetricsMeter(String tenantId) {
+        if ( StringUtil.isNullOrEmpty(tenantId) ) {
+            return null;
+        }
+        return Metrics.meter(Instrumentation.class, "tenants", tenantId, "Data Points Ingested");
```

Comment:
I just want to double check. The meter is created only once per tenant right? Eventhough this method is called several times?